app/helpers/lists_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ListsHelper
-  def format_list_name(list)
-    list.updated_at.localtime.strftime('%e %b %Y, %l:%M:%S %P')
+  def format_time(timestamp)
+    timestamp.localtime.strftime('%e %b %Y, %l:%M:%S %P')
   end
```

Comment:
So another common way this is done in rails is through the `l` helper method (short for 'localize') which is here https://guides.rubyonrails.org/i18n.html#adding-date-time-formats.

Reply:
Excellent, I refactored to use this in e41970e